hw/ip/kmac/doc/checklist.md

```diff
@@ -11,13 +11,13 @@ All checklist items refer to the content in the [Checklist.]({{<relref "/doc/pro
 
 Type          | Item                           | Resolution  | Note/Collaterals
 --------------|--------------------------------|-------------|------------------
-Documentation | [SPEC_COMPLETE][]              | Not Started | [KMAC Spec][../]
+Documentation | [SPEC_COMPLETE][]              | Done        | [KMAC Spec][../]
 Documentation | [CSR_DEFINED][]                | Done        |
 RTL           | [CLKRST_CONNECTED][]           | Done        |
 RTL           | [IP_TOP][]                     | Done        |
 RTL           | [IP_INSTANTIABLE][]            | Done        |
 RTL           | [PHYSICAL_MACROS_DEFINED_80][] | Done        |
```

Comment:
i think @sriyerg suggestion for macro is that we leave it NA if there are no memories etc.

Reply:
Revised. Thanks!